OutOfSchool/OutOfSchool.BusinessLogic/Models/Workshops/WorkshopCard.cs

```diff
@@ -52,8 +52,8 @@ public static WorkshopCard ToCard(this Workshop workshop)
         => new()
         {
             Id = workshop.Id,
-            ProviderTitle = workshop.ProviderTitle,
-            ProviderTitleEn = workshop.ProviderTitleEn,
+            ProviderTitle = workshop.Provider?.FullTitle,
```

Comment:
more details in predicate comment, i won't write it for every DTO because it is mostly critical for types that we return in lists, such as cards.

Make sure the provider is eagerly included in the request to database, so we avoid n+1 query explosion for lists